assets/settings/default.json

```diff
@@ -1399,6 +1399,8 @@
   "journal": {
     // The path of the directory where journal entries are stored
     "path": "~",
+    // What name to use for folders and journal files.
+    "date_format": "%Y/%m/%d.md",
```

Comment:
We have a configuring-zed.md file where this also needs to be added, so that zed.dev/docs gets something too.

Also, the date format: it would be nice to mention what are the options for the people.
`format` library method points at `strftime.rs` docs in `chrono` crate, https://docs.rs/chrono/latest/chrono/format/strftime/index.html 

Seems worth adding that?